compliance/Azure_long_stopped_instances/long_stopped_instances_azure.pt

```diff
@@ -0,0 +1,210 @@
+name "Azure long stopped instances"
+rs_pt_ver 20180301
+type "policy"
+short_description "Report on any instances that have been stopped for a long time with the option to Terminate them. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/compliance/azure/long_stopped_instances) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+category "Compliance"
+severity "low"
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses of the recipients you wish to notify"
```

Comment:
needs description.  see other policies for examples.

Reply:
So for this one... I looked at the other policies, and it matches... its also what was in the POL request... not sure what else you are asking for here.

Reply:
some older policies don't follow the best practice, we are updating them as we go.  new polcies parameters should have a label and description.   here is an example https://github.com/rightscale/policy_templates/blob/master/cost/scheduled_reports/scheduled_report.pt#L28